### DISCUSSION TITLE
🗳️ We want your help selecting the OpenTF Logo!

### DISCUSSION THREAD

**Discussion by coryodaniel** [opentffoundation/manifesto#403] [Polls]
Please vote on your logo choice below!

![OpenTF Logo Choices](https://github.com/opentffoundation/manifesto/assets/11587/881ab423-93a3-42ff-9eb9-4f33c9541f40)

**Comment by lorengordon** [comment]
The symbol in #1 is the best, but the font is the worst

**Reply by jacob-leger** [reply]
Agreed. I occasionally do logo design, and the first icon with the second font family and styles would be perfect.

**Comment by AJS23** [comment]
I guess number 5 would be a great choice, as the open space in the letter O gives a message of open to feedback.!

**Comment by skhelishalnev** [comment]
Agree 5 looks great other have some resemblance of other projects like tenforflow or commercial terraform 😏

**Comment by Magnitus-** [comment]
1, 3 or 5 for me, but I'm torn between those. Will have to spend more time contemplating the options after work.

**Reply by spaceCabbage** [reply]
3 is super clean
